cns/service/main.go

```diff
@@ -795,6 +801,35 @@ func main() {
 		}
 	}
 
+	type CNSService struct {
+		pb.UnimplementedCNSServiceServer
+		Logger *zap.Logger
+	}
+
+	// Define gRPC server settings
+	settings := grpc.GrpcServerSettings{
+		IPAddress: "::",
+		Port:      8080,
+	}
```

Comment:
We should eventually add move these parameters to the CNS config

Reply:
I have defined those parameters in the CNS Config instead of the main file.

Reply:
I don't mind if the root CNS config is composed of imported subconfigs. It probably it easier to understand what the options are of the subpackage if its config lives with it. But it is important that the package only ever references its local config, not the root CNS config, so we don't have a cyclical import issue.